godot-macros/src/docs.rs

```diff
@@ -146,12 +146,26 @@ fn xml_escape(value: String) -> String {
     result
 }
 
+/// Utility function that, given a line from a code comment, returns the line
+/// without the first leading space if it exists. This space is commonly used
+/// to separate the comment from the `///` that precedes it, but interferes
+/// with the formatting of the resulting Markdown.
+fn remove_initial_space(comment_line: &str) -> String {
+    if comment_line.starts_with(" ") {
+        comment_line[1..].to_string()
+    } else {
+        comment_line.to_string()
+    }
+}
```

Comment:
can't we just do this to avoid allocating here?
```suggestion
fn remove_initial_space(comment_line: &str) -> &str {
    if comment_line.starts_with(" ") {
        &comment_line[1..]
    } else {
        comment_line
    }
}
```

Reply:
Resolved.